pkgs/applications/video/lbry/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchurl
+, stdenv
+, appimageTools
+, ffmpeg
+, gtk3
+, makeWrapper
+, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lbry-desktop";
+  version = "0.50.2";
+
+  # Fetch and extract the AppImage
+  src = appimageTools.extract {
+    name = pname;
+    src = fetchurl {
+      url = "https://github.com/lbryio/lbry-desktop/releases/download/v${version}/LBRY_${version}.AppImage";
+      # Gotten from latest-linux.yml
+      sha512 = "br6HvVRz+ybmAhmQh3vOC5wgLmOCVrGHDn59ueWk6rFoKOCbm8WdmdadOZvHeN1ld2nlvPzEy+KXMOEfF1LeQg==";
+    };
+  };
+
+  buildInputs = [ ffmpeg gtk3 ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+
+    # place the extracted AppImage source in lib
+    LBRY_PATH="$out/lib/${pname}"
+    mkdir -p $LBRY_PATH
+    cp -r $src/* $LBRY_PATH
+
+    # executable wrapper
+    mkdir -p $out/bin
+    makeWrapper $LBRY_PATH/lbry $out/bin/lbry \
+      --prefix XDG_DATA_DIRS : $GSETTINGS_SCHEMAS_PATH \
+      --prefix PATH : ${ffmpeg}/bin:${xdg_utils}/bin
+
+    # desktop file/icon
+    mkdir -p $out/share/applications $out/share/icons/hicolor/256x256/apps
+    ln -s $LBRY_PATH/lbry.png $out/share/icons/hicolor/256x256/lbry.png
+    ln -s $LBRY_PATH/lbry.desktop $out/share/applications/lbry.desktop
+    substituteInPlace $LBRY_PATH/lbry.desktop \
+      --replace AppRun $out/bin/lbry
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A browser and wallet for LBRY, the decentralized, user-controlled content marketplace.";
+    longDescription = ''
+      The LBRY app is a graphical browser for the decentralized content marketplace provided by the LBRY protocol. It is essentially the lbry daemon bundled with a UI using Electron.
+    '';
+    license = licenses.mit;
+    homepage = "https://lbry.com/";
+    downloadPage = "https://lbry.com/get/";
+    changelog = "https://github.com/lbryio/lbry-desktop/blob/master/CHANGELOG.md";
+    maintainers = with maintainers; [ enderger ];
+    platforms = [ "x86_64-linux" ];
+  };
+}
+
+
```

Comment:
```suggestion

```
One superfluous line at end of file

Reply:
Ah, looks like that's another on-save script I need to add to my vimrc. I'll fix that.